pkg/docker/mover.go

```diff
@@ -17,6 +17,11 @@ type ImagePusher interface {
 	PushImage(ctx context.Context, image string, endpoint string) error
 }
 
+type ImageTaggerPusher interface {
+	ImagePusher
+	TagImage(ctx context.Context, image string, endpoint string) error
```

Comment:
why dont we just add this in `ImagePusher`?

Reply:
Because `ImagePusher` is supposed to only push (hence the name)
I can delete `ImagePusher` and have both methods defined in `ImageTaggerPusher` if you prefer

Reply:
yea lets remove `imagePusher` then. the level in interface is not that necessary imo